subjects/min_and_max/README.md

```diff
@@ -0,0 +1,45 @@
+## min_and_max
+
+### Instructions
+
+Create a **function** named `min_and_max` that receives three `i32` and returns a `tuple` with the minimum and the maximum number received as input.
+
+```rust
+pub fn min_and_max(nb_1: i32, nb_2: i32, nb_3: i32) -> (i32, i32) {
+}
+```
+
+### Usage
+
+Here is a program to test your function
+
+```rust
+use min_and_max::min_and_max;
+
+fn main() {
+    let nb_1 = 9;
+    let nb_2 = 4;
+    let nb_3 = 2;
+    let (min, max) = min_and_max(nb_1, nb_2, nb_3);
```

Comment:
maybe we can get rid of the 3 variables and write the arguments inline.
I think it's better if our examples stays as minimal as possible, and students can infer information instead of just read it.